src/test/expected/binop/main.rs

```diff
@@ -56,19 +56,22 @@ fn ibxor_test(a: i32, b: i32, correct: i32, wrong: i32) {
 }
 
 fn main() {
```

Comment:
IS this to prevent one failure from hiding others?

Reply:
Yes. Otherwise everything passed `iadd_test()` becomes unreachable.